doc/articles/migrating-to-uno-sdk.md

```diff
@@ -0,0 +1,195 @@
+
+```json
+{
+  "msbuild-sdks": {
+    "Uno.Sdk": "{Current Version of Uno.WinUI/Uno.Sdk}",
+    "Microsoft.Build.NoTargets": "3.7.56"
+  }
+}
+```
+
+> [!NOTE]
+> The Microsoft.Build.NoTargets Sdk has been used for the last several versions in the Uno Templates. Depending on which version of the templates this may have been called `{YourProject}.Base` or `{YourProject}.Shared`. With the introduction of the global.json you may want to update the Sdk in the csproj to remove the version and use the version provided by the global.json instead. This makes it easier to manage your Sdk versions centrally.
+
+## Using the Uno.Sdk
+
+Modern .NET Projects use a Sdk standard. By default your project's typically will target the `Microsoft.NET.Sdk`. Some projects such as the Wasm and Server projects in the Uno Templates will target `Microsoft.NET.Sdk.Web`. As previously mentioned the `.Shared` project targets the `Microsoft.Build.NoTargets` Sdk. The difference between these Sdk's is the `Microsoft.Build.NoTargets` much like the `Uno.Sdk` are shipped and versioned on NuGet.org while the first two are shipped and installed as part of the larger .NET Sdk/Runtime itself.
+
+To update your projects you simply need to replace `Microsoft.NET.Sdk` with `Uno.Sdk`. You will do this in the projects that you would reference Uno from. As an example let's say that you called your project `AwesomeProject`, the following projects would need to have this updated:
+
+- AwesomeProject
+- AwesomeProject.Mobile
+- AwesomeProject.Skia.Gtk
+- AwesomeProject.Skia.Linux.FrameBuffer
+- AwesomeProject.Skia.WPF
+- AwesomeProject.Windows
+
+```xml
+<Project Sdk="Uno.Sdk">
+```
+
+We will also need to update the `AwesomeProject.Wasm` project however this needs to be done slightly differently currently. Both `Uno.Sdk` and `Microsoft.NET.Sdk.Web` will actually load the `Microsoft.NET.Sdk`. However `Microsoft.NET.Sdk.Web` does not currently check to see if `Microsoft.NET.Sdk` has already been loaded. As a result you will need to do the following:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.Web">
+  <Sdk Name="Uno.Sdk" />
+```
+
+## Uno Version
+
+The core libraries that ship from the main Uno repo (these are versioned together with Uno.WinUI), should now be controlled by the version of the Uno.Sdk that you are using. If you are using Central Package Management (there is a `Directory.Packages.props` in your solution) this will be very easy as you simply need to open the `Directory.Packages.props`. If you are not using Central Package Management you will need to open the various csproj files that have a Package Reference to one of the Uno NuGet's. You should do a Find/Replace for the version of Uno that you are using and replace it with `$(UnoVersion)`. The `$(UnoVersion)` property is set by the Uno.Sdk. To update Uno in the future simply update the version of the `Uno.Sdk` that you are targeting in the global.json as shown above.
```

Comment:
```suggestion
The core libraries that ship from the main Uno repo (these are versioned together with Uno.WinUI.*), can now be controlled by the version of the Uno.Sdk that you are using. This common version is available through the `$(UnoVersion)` property set by the Uno.Sdk. 

If you are using Central Package Management (there is a `Directory.Packages.props` in your solution) this will be very easy as you simply need to open the `Directory.Packages.props`. 

If you are not using Central Package Management you will need to open the various csproj files that have a Package Reference to one of the Uno NuGet's. You should do a Find/Replace for the version of Uno that you are using and replace it with `$(UnoVersion)`. 

To [update Uno Platform in the future](xref:Uno.Development.UpgradeUnoNuget) simply update the version of the `Uno.Sdk` that you are targeting in the `global.json` as shown above.
```